deploy.cmd

```diff
@@ -103,9 +103,9 @@ IF EXIST "%DEPLOYMENT_SOURCE%\package.json" (
 IF EXIST "%DEPLOYMENT_SOURCE%/.angular.json" (
 echo Building App in %DEPLOYMENT_SOURCE%…
 pushd "%DEPLOYMENT_SOURCE%"
-call :ExecuteCmd !NPM_CMD! run build
+:: call :ExecuteCmd !NPM_CMD! run build
 :: If the above command fails comment above and uncomment below one
-:: call ./node_modules/.bin/ng build –prod
+call ./node_modules/.bin/ng build –prod
```

Comment:
This would need to be `call .\node_modules\.bin\ng build -prod` (backslashes not forward slashes).  However, I don't think this will make any difference, because the `ng build` command was started successfully but failed with the error `Cannot read property 'config' of null`

Reply:
yep, it didn't work :(